library/consul

```diff
@@ -1,13 +1,13 @@
-Maintainers: Preetha Appan <preetha@hashicorp.com> (@preetapan)
+Maintainers: Consul Team <consul@hashicorp.com>
 
-Tags: 1.3.0, latest
+Tags: 1.3.1
```

Comment:
After we get this 1.3.1 bump in, will this section be going away?  Removing it here will remove it from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding it on the official build server), but the tag will still be available to users who want it.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

Reply:
Hey @tianon, yep this will go away once it's pushed - we just patched something that prevented upgrade in the same week as releasing 1.4.0 so need the image to be available. Per the docs this can be considered a "backfilled version" although we technically release 1.3.1 right before 1.4.0.